drivers/dac/ltc2672/ltc2672.h

```diff
@@ -0,0 +1,251 @@
+#define LTC2672_16BIT_RESO   			65535
+#define LTC2672_12BIT_RESO   			4095
+#define LTC2672_TOTAL_CHANNELS 			5
+#define LTC2672_MAX_CONFIG_MASK  		15
+#define LTC2672_MAX_TOGGLE_MASK  		31
+
+enum ltc2672_commands {
+	LTC2672_CODE_TO_CHANNEL_X,
+	LTC2672_PWRUP_UPD_CHANNEL_X,
+	LTC2672_CODE_TO_CHANNEL_X_PWRUP_UPD_CHANNEL_ALL,
+	LTC2672_CODE_PWRUP_UPD_CHANNEL_X,
+	LTC2672_PWRDWN_CHANNEL_X,
+	LTC2672_PWRDWN_DEV,
+	LTC2672_SPAN_TO_CHANNEL_X,
+	LTC2672_CNFG_CMD,
+	LTC2672_CODE_TO_CHANNEL_ALL,
+	LTC2672_PWRUP_UPD_CHANNEL_ALL,
+	LTC2672_CODE_PWRUP_UPD_CHANNEL_ALL,
+	LTC2672_MON_MUX,
+	LTC2672_TOGGLE_SEL,
+	LTC2672_TOGGLE_GLBL,
+	LTC2672_SPAN_TO_CHANNEL_ALL,
+	LTC2672_NO_OP
+};
+
+/* LTC2672 Command Generation */
+#define LTC2672_COMMAND32_GENERATE(comm, add, dat) \
+		(0xFF << 24) | (comm << 20) | (add << 16) | (dat)
+
+#define LTC2672_COMMAND24_GENERATE(comm, add, dat) \
+		(comm << 20) | (add << 16) | (dat)
+
+#define LTC2672_SPAN_SET(span_code) 	 LTC2672_16_DONT_CARE | span_code
+
+#define LTC2672_MUX_SET(mux_code) 	 	 LTC2672_MUX_DONT_CARE | mux_code
+
+#define LTC2672_MUX32_GENERATE(comm, dat) \
+		(0xFF << 24) | (comm << 20) | (dat)
+
+#define LTC2672_MUX24_GENERATE(comm, dat) \
+		(comm << 20) | (dat)
+
+/* Device Family */
+enum ltc2672_device_id {
+	LTC2672_12,
+	LTC2672_16
+};
+
+/* DAC Channels */
+enum ltc2672_dac_ch {
+	LTC2672_DAC0,
+	LTC2672_DAC1,
+	LTC2672_DAC2,
+	LTC2672_DAC3,
+	LTC2672_DAC4
+};
+
+/* Output Range */
+enum ltc2672_out_range {
+	LTC2672_OFF, // Off mode
+	LTC2672_50VREF, // 3.125mA
+	LTC2672_100VREF, // 6.25mA
+	LTC2672_200VREF, //12.5mA
+	LTC2672_400VREF, // 25mA
+	LTC2672_800VREF, // 50mA
+	LTC2672_1600VREF, // 100mA
+	LTC2672_3200VREF,  // 200mA
+	LTC2672_VMINUS_VREF, // Fixed, -80mA
+	LTC2672_4800VREF = 0XF // 300mA
+};
+
+/* Multiplexer Command Codes */
+enum ltc2672_mux_commands {
+	LTC2672_MUX_DISABLED,
+	LTC2672_MUX_IOUT0,
+	LTC2672_MUX_IOUT1,
+	LTC2672_MUX_IOUT2,
+	LTC2672_MUX_IOUT3,
+	LTC2672_MUX_IOUT4,
+	LTC2672_MUC_VCC,
+	LTC2672_MUX_VREF = 0x08,
+	LTC2672_MUX_VREF_LO,
+	LTC2672_MUX_DIE_TEMP,
+	LTC2672_MUX_VDD0 = 0x10,
+	LTC2672_MUX_VDD1,
+	LTC2672_MUX_VDD2,
+	LTC2672_MUX_VDD3,
+	LTC2672_MUX_VDD4,
+	LTC2672_MUX_VMINUS = 0X16,
+	LTC2672_MUX_GND,
+	LTC2672_MUX_VOUT0,
+	LTC2672_MUX_VOUT1,
+	LTC2672_MUX_VOUT2,
+	LTC2672_MUX_VOUT3,
+	LTC2672_MUX_VOUT4
+};
+
+/* Faults */
+enum ltc2672_faults {
+	LTC2672_OPEN_OUT0, // Open circuit CH0
+	LTC2672_OPEN_OUT1, // Open circuit CH0
+	LTC2672_OPEN_OUT2, // Open circuit CH0
+	LTC2672_OPEN_OUT3, // Open circuit CH0
+	LTC2672_OPEN_OUT4, // Open circuit CH0
+	LTC2672_OVER_TEMP, // Over-temperature (T > 175 deg C)
+	LTC2672_UNUSED, // Unused fault register bit
+	LTC2672_INV_LENGTH, // Invalid SPI Length (len != 24 or 32 * n)
+};
+
+/**
+ * @struct ltc2672_dev
+ * @brief Structure holding ltc2672 descriptor.
+ */
+struct ltc2672_dev {
+	/* SPI descriptor */
+	struct no_os_spi_desc *comm_desc;
+	/* Device Variant indicator */
+	enum ltc2672_device_id active_device;
```

Comment:
`active_device` is kinda long...
I know this is semantics, but `id` would make more sense.

Reply:
Changed active_device property to id as suggested on recent push.